src/styles/main.scss

```diff
@@ -0,0 +1,17 @@
+@import "utils/keyframes";
+@import "utils/variables";
+@import "blocks/clock.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  width: 100vw;
```

Comment:
Generally don't set width to 100vw, if you had a vertical scroll this would cause an overflow and horizontal scroll which we mostly need to avoid